life_expectancy/cleaning.py

```diff
@@ -0,0 +1,136 @@
+"""Module for cleaning life expectancy data."""
+import argparse
+import pandas as pd
+
+def main(region: str) -> None:
+    """Main function."""
+    full_data = _load_data('life_expectancy/data')
+    region_data = _clean_data(full_data, region)
+    _save_data_csv('life_expectancy/data', region_data, region)
+
+def _clean_data(data: pd.DataFrame, region: str) -> pd.DataFrame:
+    """ Clean the data and return only a given region.
+
+    Args:
+        region: the name of the region to clean
+    """
+    full_data = (
+        data
+        .pipe(_melt_table)
+        .pipe(_clean_year_value_cols)
+        .pipe(_clean_region_col)
+    )
+    region_data = _select_region(full_data, region)
+
+    return region_data
+
+def _load_data(data_path: str) -> pd.DataFrame:
+    """Load the data.
+
+    Returns:
+        data (pd.DataFrame): The data.
+    """
+    return pd.read_table(f'{data_path}/eu_life_expectancy_raw.tsv')
+
+def _melt_table(data: pd.DataFrame) -> pd.DataFrame:
+    """Melt the table to long format.
+    
+    Args:
+        data (pd.DataFrame): The data.
+        
+    Returns:
+        data (pd.DataFrame): The data in long format.
```

Comment:
I think that the type hints are unnecessary here since you did it already above

Reply:
yes, as before. docstring auto gen.